modulus/datapipes/healpix/data_modules.py

```diff
@@ -790,3 +791,313 @@ def test_dataloader(self, num_shards=1, shard_id=0) -> DataLoader:
         )
 
         return loader, sampler
+
+
+class CoupledTimeSeriesDataModule(TimeSeriesDataModule):
```

Comment:
Do all the utils in this file have tests?

Reply:
Yes, CoupledTimeSeriesDataModule has tested.